templates/iam_policy/ip_lock_checker_policy.json.tpl

```diff
@@ -0,0 +1,24 @@
+{
+  "Statement": [
+    {
+      "Action": "events:PutEvents",
+      "Effect": "Allow",
+      "Resource": "arn:aws:events:eu-west-2:${account_id}:event-bus/default",
+      "Sid": "readSecretsManager"
```

Comment:
Is this name correct?

Reply:
Ah no, copy and paste error. I've changed it.